app/presenters/tree_node_builder.rb

```diff
@@ -30,8 +30,8 @@ def self.build(object, parent_id, options)
     builder.build
   end
 
-  def self.build_id(object, parent_id, options)
-    builder = new(object, parent_id, options)
+  def self.build_id(object, options)
+    builder = new(object, nil, options)
     builder.build_id
```

Comment:
might as well remove this useless temporary variable while you're in here (if you have to come back around to changing the code)